src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -101,23 +141,33 @@
                                             j++;
                                         }
                                     }
-
+                                    //should only be reached after for loop is done
                                     instanceTransactionViewModels.get(0).setValidity(tangle, 1);
                                     transactions.add(instanceTransactionViewModels);
-                                } else {
+                                }
+                                //bundle hash was not verified
```

Comment:
```suggestion
                                //bundle hash verification failed
```
or something like that - it's not that it didn't run verification, but it failed.

Reply:
:+1: